socialite/src/components/StackOverflow/StackOverflow.js

```diff
@@ -26,35 +31,35 @@ function StackOverflow(props){
   var post_list = data? data.getPosts : "";
 
 	return (
-            <>
+          <>
       		<div className="container">
-      			<div className="authenticate-nav">
-                              <div className="a-nav-right">
-                                    <button className="rounded" onClick={logUserOut}>LOGOUT</button>
-                              </div>
-                        </div>
-
-                        <div className="feature-display">
-                              <div className="subsection-header"> Top Posts </div>
-                                    <div className="row">
-                                     {post_list && post_list.map(post => ( 
-                                        <div className="col-lg-12">
-                                          <div className="friend" onClick={() => showIssue(post['id'])}>
-                                            <div className="friend-content">
-                                                <strong>Created By: </strong>  {post['email']}
-                                                <br />
-                                                Body:{post['body']}
-                                          </div>
-                                        </div>
-                                    </div>
-                                     ))} 
-                                    </div>
-                              </div>
+      			  <div className="authenticate-nav">
+                  <div className="a-nav-right">
+                      <button className="rounded" onClick={dashboard}>DASHBOARD</button>
+                      &nbsp;&nbsp;
```

Comment:
use margin classes instead of nbsp

Reply:
Fixed the margin issues with the merge conflicts